files/build_templates/sonic_debian_extension.j2

```diff
@@ -353,6 +353,14 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get purge
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get clean -y
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get autoremove -y
 
+# install kernel dump utility
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get install -y kdump-tools || true
+sudo sed -i 's/\/MODULES=dep\//\/MODULES=most\//' $FILESYSTEM_ROOT/etc/kernel/postinst.d/kdump-tools
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get install -y kdump-tools
+{% if sonic_kdump_enable == "y" %}
```

Comment:
if-block should contain the upper 3 statements?

Reply:
@qiluo-msft 

only install crash tool  when kdump_enable

Reply:
I am wondering should we install kdump-tools and crash only when kdump_enable


---
In reply to: [338329974](https://github.com/Azure/sonic-buildimage/pull/3276#discussion_r338329974) [](ancestors = 338329974)

Reply:
totally understood your wondering, but this is suggested by guohan, install kdump-tools by default but install crash tool when kdump_enabled

Reply:
thanks a lot for the review and approval:)